doc/_info_/11logo.order

```diff
@@ -0,0 +1,481 @@
+PENDOWN
+
+     Takes no input. It causes the turtle to draw a line when it moves.
+
+PENUP
+
+     Takes no inputs. It causes the turtle to not draw a line when in
+     moves.
+     
+PRINT
+
+     Takes one input, which evaluates to either a word or a list.
+     Prints out the evaluated input on the console.
+          ?PRINT SUM 4 4
+	  8
+	  ?PRINT [SUM 4 4]
+	  [SUM 4 4]
+     Please note that procedure PRINT in that version has unusual behavior
+     and prints the top-level brackets around a list.
+	  
+PRODUCT
+
+     Takes two numeric inputs and outputs their product.
+          PRINT PRODUCT 2 3 prints 6.
+	  
+QUOTIENT
+
+     Takes two numeric inputs and outputs their quotient. (The first
+     input is divided by the second).
+               PRINT QUOTIENT 4 2 prints 2.
+	       
+REQUEST
+
+     No inputs. When encountered in the execution of a procedure,
+     causes LOGO to pause and wait for a line to be typed in. This
+     input is treated as a list and printed on the console.
+     
+RIGHT
+
+     Takes one numeric input. Commands the turtle to turn to the right
+     (clockwise) the number of degrees which you give as an input.
+               RIGHT 30 tells the turtle to turn 30 degrees to the right.
+RUG
+
+     probably jumps to the PDP-11 debugger. We have it (RUG; AR >),
+     but how to get both that and 11LOGO running we don't know yet.
+
+SENTENCE
+
+     Takes two inputs. If both are lists it puts the elements of the
+     lists together to make a single list and outputs that list. If
+     either of its inputs is not a list it first changes the input to
+     a one-element list and then proceeds as above,
+               SENTENCE [FOO] [FOO BAR] outputs [FOO FOO BAR]
+	       SENTENCE [WHAT IS YOUR] "NAME? outputs [WHAT IS YOUR NAME?]
+
+SETHEADING
+
+     Takes one numeric input. Specifies the direction (in degrees) in
+     which the display turtle points. Zero is straight up.
+     
+SETTURTLE
+
+     Takes one input which is a list of 3 numbers. This input assigns,
+     in order, the x-coordinate, the y-coordinate, and the heading of the
+     display turtle.
+     
+SETX
+
+     Takes one numeric input. Moves the turtle horizontally to the
+     specified coordinate.
+     
+SETXY
+
+     Takes two numeric inputs. Moves the display turtle to the position
+     designated.
+     
+SETY
+
+     Takes one numerical input. It moves the display turtle vertically
+     to the specified coordinate.
+     
+SHOW
+
+     Takes one input, the name of a procedure, and prints the text of
+     the procedure on the console. The name of a procedure should be 
+     qoted.
+               SHOW "PROCEDURE
+     
+SHOWTURTLE
+
+     Takes no inputs. It tells the computer to put the display turtle
+     (a little triangle) on the the display screen.
+
+SNAP
+
+     Takes no inputs. Outputs a reference to "the stuff on the display
+     screen." For example,
+               MAKE "PIC SNAP causes :PIC to refer to whatever is

```

Comment:
add newline before "causes"

Reply:
First of all I want to thank you, @eswenson1 and @atsampson for helping to make the document meet the appropriate quality standard. It would not be possible without @larsbrinkhoff and you help. 

As for the issue itself, I tried to follow the original A.I. Memo 315 text formatting style. If fact, I did not change the text unnecessarily and in this particular case it is exactly copy from A.I. Memo 315:

![aim-315](https://user-images.githubusercontent.com/42380964/45315749-d79fb280-b53d-11e8-8006-33cdd5993ac6.png)

 So, I would like to leave the original text and formatting style in the document if it is possible.